packages/@aws-cdk/aws-codepipeline-actions/test/integ.lambda-deployed-through-codepipeline.lit.ts

```diff
@@ -47,7 +47,7 @@ pipeline.addStage({
 // synthesize the Lambda CDK template, using CodeBuild
 // the below values are just examples, assuming your CDK code is in TypeScript/JavaScript -
 // adjust the build environment and/or commands accordingly
-const cdkBuildProject = new codebuild.Project(pipelineStack, 'CdkBuildProject', {
+const cdkBuildProject = new codebuild.PipelineProject(pipelineStack, 'CdkBuildProject', {
```

Comment:
Why is this change needed?

Reply:
It's because of the validation that you asked to add in the previous revision (checking that the source of the project is pipeline: https://github.com/aws/aws-cdk/pull/4183#discussion_r332232965 ).